openmetadata-ui/src/main/resources/ui/src/components/common/RichTextEditor/CustomHtmlRederer/CustomHtmlRederer.tsx

```diff
@@ -61,6 +68,22 @@ const getHTMLTokens = (node: MdNode): HTMLToken[] => {
   ];
 };
 
+export const CodeMirrorLanguageAliases: Readonly<Record<string, string>> = {
+  // Mappings for C-like languages https://codemirror.net/5/mode/clike/index.html
+  c: 'text/x-csrc',
+  'c++': 'text/x-c++src',
+  java: 'text/x-java',
+  csharp: 'text/x-csharp',
+  scala: 'text/x-scala',
+  kotlin: 'text/x-kotlin',
+  objectivec: 'text/x-objectivec',
+  'objectivec++': 'text/x-objectivec++',
+  // Aliases for convenience
+  js: 'javascript',
+  py: 'python',
+  cpp: 'text/x-c++src',
+};
```

Comment:
Also, for user's convenience, I have added the ability to set an alias for the language name.

Reply:
Thanks for tacking care of this @fuzmish